elixir/apps/web/test/web/live/actors/show_test.exs

```diff
@@ -338,6 +338,39 @@ defmodule Web.Live.Actors.ShowTest do
       )
     end
 
+    test "allows sending welcome email", %{
+      account: account,
+      actor: actor,
+      identity: admin_identity,
+      conn: conn
+    } do
+      Domain.Config.put_env_override(:outbound_email_adapter_configured?, true)
+      email_provider = Fixtures.Auth.create_email_provider(account: account)
+
+      other_identity =
+        Fixtures.Auth.create_identity(account: account, actor: actor, provider: email_provider)
+        |> Ecto.Changeset.change(
+          created_by: :identity,
+          created_by_identity_id: admin_identity.id
+        )
+        |> Repo.update!()
+
+      {:ok, lv, _html} =
+        conn
+        |> authorize_conn(admin_identity)
+        |> live(~p"/#{account}/actors/#{actor}")
+
+      assert lv
+             |> element("#identity-#{other_identity.id} button", "Send Welcome Email")
+             |> render_click()
+             |> Floki.find(".flash-info")
+             |> element_to_text() =~ "Welcome email sent to #{other_identity.provider_identifier}"
+
+      assert_email_sent(fn email ->
+        assert email.subject == "Welcome to Firezone"
```

Comment:
I would add assertions to make that least for `actor.name` and `account.slug` are part of the email, plus we will need a test case when IdP did not return us an `email` claim.

Reply:
Adding the assertions is no problem.

As far as the other test case, are you saying you'd like to see a test case that verifies that the "Send Welcome Email" doesn't show up when the identity doesn't have an email?

Reply:
@bmanifold yeah, also:
- that send works for IdPs and uses email from `user_info` (eg.  Google Workspace, where provider_identifier is just an ID)
- that email doesn't contain such ID if `user_info` doesn't have an email